service.go

```diff
@@ -297,3 +301,22 @@ func (s *serviceManager) serviceByID(id ServiceID) (Service, bool) {
 	}
 	return serv, true
 }
+
+// NewProtocol contains the logic of how and where a ProtocolInstance is
+// created. If the token's ServiceID is nil, then onet handles the creation of
```

Comment:
In the readme or presentations, I'm ok with "ONet", but in the code, I would stick with "onet" because 
+  it is easier to read & remember: which letter is uppercase? is it "Onet" or "ONet" or "ONET" ? 
+  to use this library, one must import "onet" and not "ONet"; it might feel strange to have two name for one thing. 

While I agree that ONet sounds more sexy and I am totally ok to show this library with such a name if we feel it's better, in the code, a more consistent and easier naming is probably better.